### PR TITLE
[Documentation][Cookbook] How to render a menu of taxons?

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| BC breaks? | no |
| Related tickets | partially #5275 |
| License | MIT |

### CODE REVIEW

File: docs/cookbook/taxons-menu.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+How does it look like?
+----------------------
+
+That's a menu that you will find on the default Sylius homepage:
+
+.. image:: ../_images/taxons_menu.png
+    :align: center
+
+How to use it?
```
Comment (NeverResponse) [inline docs/cookbook/taxons-menu.rst]: more of a `do it` rather than `use it`

File: docs/cookbook/taxons-menu.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
+How to render a menu of taxons?
+===============================
+
+The way of rendering a menu of taxons that you know form Sylius default homepage view is a supereasy reusable action.
```
Comment (NeverResponse) [inline docs/cookbook/taxons-menu.rst]: `from` 😭  besides this sentence is quite a trainwreck

File: docs/cookbook/taxons-menu.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+That's a menu that you will find on the default Sylius homepage:
+
+.. image:: ../_images/taxons_menu.png
+    :align: center
+
+How to use it?
+--------------
+
+You can render such a menu wherever you have access to a ``category`` variable in the view, but also anywhere else.
```
Comment (NeverResponse) [inline docs/cookbook/taxons-menu.rst]: So... wherever you'd like?

Comment (TheMadeleine) [inline docs/cookbook/taxons-menu.rst]: Actually yes. That's explained below.

**Comment by pjedrzejewski** [comment]
Thanks Magda!

**Pull request merged**